tools/bridge/tests/conftest.py

```diff
@@ -24,3 +27,50 @@ def w3_foreign(tester_foreign):
 def w3_home(tester_home):
     provider = EthereumTesterProvider(tester_home)
     return Web3(provider)
+
+
+@pytest.fixture
+def token_transfer_event_queue():
+    return Queue()
+
+
+@pytest.fixture(scope="session")
```

Comment:
Any reason this has `scope="session"`?

Reply:
Nope. Just try to add it where possible to probably save some time. I remove it.

Reply:
:heavy_check_mark: